2021-09-30 02:37:58.484 [error] <0.112.0>@merge_dets_env:merge:32 gen_server merge_main_proc terminated with reason: bad argument in call to dets:delete_all_objects(env_file) line 261 in merge_dets_env:merge/1 line 32
2021-09-30 02:37:58.488 [error] <0.112.0>@merge_dets_env:merge:32 CRASH REPORT Process <0.112.0> with 1 neighbours exited with reason: bad argument in call to dets:delete_all_objects(env_file) line 261 in merge_dets_env:merge/1 line 32
2021-09-30 02:37:58.488 [error] <0.110.0>@merge_dets_env:merge:32 Supervisor merge_sup had child merge_context started with merge_context:start_link() at <0.111.0> exit with reason bad argument in call to dets:delete_all_objects(env_file) line 261 in merge_dets_env:merge/1 line 32 in context child_terminated
